### PR TITLE
latest immuns, paths to work on windows, delivery tweaks

### PR DESCRIPTION
**Pull request by theism**
getting brian's changes from latest updates to work on windows
latest immun changes to run for last analysis (no need to review, just committing latest)

### CODE REVIEW

File: delivery.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def _process_metric(df, col, locations, totalname=''):
         loc = locations[count:]
         new_agg = (df.groupby(loc)[sum_col].sum() / df.groupby(loc)[total_col].sum()).reset_index()
         new_agg.columns = loc + [pct_col]
-        df = pd.merge(df, new_agg, how='left', on=loc)
+        df = pd.merge(df.reset_index(), new_agg, how='left', on=loc).set_index(locations)
```
Comment (bderenzi) [inline delivery.py]: Curious why this change had to happen? Merging a DF shouldn't be OS-dependent

File: delivery.py [hunk 2]
```diff
@@ -107,7 +107,7 @@ def _col_names_by_metric(col, locations, totalname):
     return cols
 
 # each indicator gets its own file
-have_del_block_df[locations + _col_names_by_metric('home_delivery', locations, 'delivery')].to_csv('overall_home_deliveries.csv')
-have_del_block_df[locations + _col_names_by_metric('hospital_delivery', locations, 'delivery')].to_csv('overall_hospital_deliveries.csv')
-have_del_block_df[locations + _col_names_by_metric('caesarean_delivery', locations, 'delivery')].to_csv('overall_caesarean_deliveries.csv')
+have_del_block_df[_col_names_by_metric('home_delivery', locations, 'delivery')].to_csv('overall_home_deliveries.csv')
```
Comment (bderenzi) [inline delivery.py]: I assume this change is because of the `reset_index()` / `set_index()` calls. On line `72`, I do a `reset_index()`, so `locations` shouldn't be part of the index (and thus need to be included in the slice. I'm happy with either, just curious why this was causing an issue

File: sample.settings.py [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+# Useful paths assuming relative path structure of
+# ROOT/Code/this file
+# ROOT/Data/some_data_dir/data_files
+# ROOT/Results/some_output_dir/output_files
+
+ROOT_FOLDER = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+# Use join for cross platform support
+DATA_DIR = os.path.join(ROOT_FOLDER, 'Data')
+OUTPUT_DIR = os.path.join(ROOT_FOLDER, 'Results')
```
Comment (bderenzi) [inline sample.settings.py]: Awesome, thanks. I realized later I should be using `os.path.join` for all of these for OS-independence

File: delivery.py (hunk 1 above)
Comment (theism) [reply delivery.py]: this merge threw an error on having the right columns to merge on.  i can keep in mind the actual errors in the future, but I think new_agg had the locations as columns instead of indices, and this made that explicit during the merge

File: delivery.py (hunk 2 above)
Comment (theism) [reply delivery.py]: that makes sense - I think the df had locations in the indices, hence the change I made when it errorred on my machine.
